data/wanderer/wanderers start.txt

```diff
@@ -179,67 +183,388 @@ mission "Wanderers: Hai Diplomat"
-			`	"Good to see you again Captain <last>, I have arranged for some refreshments so please, come and tell me all about these new aliens you have discovered!"`
-			`	Her exuberance is catching and after recent stressful events you see a new side to her as you relay your experience with the bird-like aliens. You realize that opportunities like this are why she became a diplomat in the first place and that this is, to some degree, a dream come true.`
-			`	"So in conclusion, it seems you need a translator," she says with a wry smile.`
+			action
+				event "A Hai professor is available to translate" 7
+			`As you approach Hai-home, you contact the Hai government and inform them that you have found new aliens to the north. You submit a request for ambassadorial assistance to help you communicate with the aliens, who seem to speak the Hai language. They immediately direct you to a government hangar in a compound called "Directorate of Intra-Hai Affairs." A Hai approaches you in an unfamiliar uniform.`
+			# Hai Intelligence officer subtly suggests they already know what's happening, and would have done
+			# something about by now if it were an opportune time.
+			`	"I received word you met some troubled avians to the north. What is your intention?"`
+			choice
+				`	"They speak the Hai language, and I need an interpreter."`
+					goto response
+				`	"I'd like to make first contact with their species."`
+					goto response
+				`	"Wait a minute. I never mentioned they were avians. Why do you say they're troubled?"`
+			`	The uniformed Hai ignores your question and asks again, "What is your intention, Captain <last>?"`
+			choice
+				`	"They speak the Hai language, and I need an interpreter."`
+				`	"I'd like to make first contact with their species."`
+			label response
+			branch election
+				or
+					has "Hai Reveal [A11] Sound the Alarm: active"
+					has "Hai Reveal [A12] Into the Lion's Den: active"
+			branch stagnant
+				not "Hai Reveal [A11] Sound the Alarm: offered"
+			# We get here if the election has ended, but the Hai-human resolution hasn't been announced
+			branch chaos
+			label election
+			`	"Need I remind you we are in the middle of an election? Barring a true emergency, our government can take no major action until the election ends. That is our law, and it is an inviolate one.`
+				goto chaos
+			label stagnant
+			`	"With the problems in human space, the Hai are always at the risk of being the recipient of an unmanageable wave of human refugees. At the same time, we struggle to prevent infighting in our own species from escalating to civil war. Now is not the time for a first contact.`
```

Comment:
```suggestion
			`	"With the problems in human space, the Hai are always at the risk of being the recipient of an unmanageable wave of human refugees. At the same time, we struggle to prevent infighting in our own species from escalating to civil war. Now is not the time for a first contact.”`
```